### PR TITLE
Merging jhon/langtonant into dev/langtonant branch

### PR DESCRIPTION
**Pull request by Alricarius**
inserting jhon's folder with the code of langton ant solution

### CODE REVIEW

File: katas/langtonant/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 The purpose of this kata is to implement a program that allow us to represent a Langton's Ant behavior. Langton's ant is a two-dimensional universal Turing machine with a very simple set of rules but complex emergent behavior, read mode abour it [here](https://en.wikipedia.org/wiki/Langton%27s_ant).
 
 ## Rules
-The Langnton's Ant consists of a group of squares on a plane (grid) colored variously either black or white. We arbitrarily identify one square as the “ant”. The ant can travel in any of the four cardinal directions at each step it takes. The “ant” moves according to the rules below:
+The Langnton's Ant consists of a group of squares on a plane (grid) colored variously either black or white. We arbitrarily identify one square as the “ant”. The ant can travel in any of the four cardinal orientations at each step it takes. The “ant” moves according to the rules below:
```
Comment (Richardson94) [inline katas/langtonant/README.md:6]: ℹ️ this change will generated furutre conflicts for future merges, i recommend to change for old announcement,. thas all
-✅ The lint task hasnt be problems
-✅ The package conflict has been resolved
-✅  The Kata's code looks funtional

**Review by Richardson94 — DISMISSED** [review]

**Comment by Alricarius** [comment]
solved merge conflicts...

**Review by Richardson94 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -5,6 +5,9 @@
   "type": "module",
   "scripts": {
     "langant:jose": "node katas/langtonant/jose/main.js",
+    "langtonant:jhon": "node katas/langtonant/jhon/main.js",
+    "langtonant:jhontest": "node katas/langtonant/jhon/board.test.js",
+    "langtonant:jhontestmodulo1": "node katas/langtonant/jose/modulo1.test.js",
```
Comment (J0c4) [inline package.json:10]: Maybe this script could be removed, is it needed?

**Review by J0c4 — APPROVED** [review]

**Comment by J0c4** [comment]
Changes looks good, please fix merge conflicts so it can be merged

**Comment by Alricarius** [comment]
merge conflict solved, ready to merge

**Review by J0c4 — APPROVED** [review]

**Pull request merged**